client/wfnews-war/src/main/angular/src/app/components/saved/saved.component.html

```diff
@@ -58,12 +58,37 @@
                         </div>
                     </div>
 
+                    <div *ngIf="location.dangerRatings && location.dangerRatings.length > 0">
```

Comment:
Some duplication here. You could likely just have the "if" on a set of img tags and not duplicate the rest